src/examples/youbot_arm_vel_test.cpp

```diff
@@ -0,0 +1,141 @@
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU Lesser General Public License LGPL and the BSD license for more details.
+*
+* You should have received a copy of the GNU Lesser General Public
+* License LGPL and BSD license along with this program.
+*
+******************************************************************************/
+
+#include <iostream>
+#include <assert.h>
+
+#include "rclcpp/rclcpp.hpp"
+#include "trajectory_msgs/msg/joint_trajectory.hpp"
+#include "brics_actuator/msg/cartesian_wrench.hpp"
+
+#include <boost/units/io.hpp>
+
+#include <boost/units/systems/angle/degrees.hpp>
+#include <boost/units/conversion.hpp>
+#include <boost/units/systems/si/angular_velocity.hpp>
+
+#include <iostream>
+#include <assert.h>
+
+#include "brics_actuator/msg/joint_positions.hpp"
+#include "brics_actuator/msg/joint_velocities.hpp"
+
+#include <boost/units/systems/si/length.hpp>
+#include <boost/units/systems/si/plane_angle.hpp>
+#include <boost/units/io.hpp>
+
+#include <boost/units/systems/angle/degrees.hpp>
+#include <boost/units/conversion.hpp>
+
+#include <chrono>
+#include <thread>
+
+using namespace std::chrono_literals;
+using namespace std::this_thread; 
+
+int main(int argc, char * argv[])
+{
+    rclcpp::init(argc, argv);
+    auto node = rclcpp::Node::make_shared("youbot_arm_vel_test");
+    auto armPositionsPublisher = node->create_publisher<brics_actuator::msg::JointVelocities> ("arm_1/arm_controller/velocity_command", 1);
+    auto gripperPositionPublisher = node->create_publisher<brics_actuator::msg::JointVelocities> ("arm_1/gripper_controller/velocity_command", 1);
+
+    double readValue;
+    static const int numberOfArmJoints = 5;
+    static const int numberOfGripperJoints = 2;
+
+    rclcpp::WallRate loop_rate(50ms);
+
+    while (rclcpp::ok()) {
+        brics_actuator::msg::JointVelocities command;
+        std::vector <brics_actuator::msg::JointValue> armJointPositions;
+        std::vector <brics_actuator::msg::JointValue> gripperJointPositions;
+
+        armJointPositions.resize(numberOfArmJoints); //TODO:change that
+        gripperJointPositions.resize(numberOfGripperJoints);
+
+        std::stringstream jointName;
+
+        for (int i = 0; i < numberOfArmJoints; ++i)
+        {
+            std::cout << "Please type in velocity for joint " << i + 1 << std::endl;
+            std::cin >> readValue;
+
+            jointName.str("");
+            jointName << "arm_joint_" << (i + 1);
+
+            armJointPositions[i].joint_uri = jointName.str();
+            armJointPositions[i].value = readValue;
+
+            armJointPositions[i].unit = boost::units::to_string(boost::units::si::radians_per_second);
+            std::cout << "Joint " << armJointPositions[i].joint_uri << " = " << armJointPositions[i].value << " " << armJointPositions[i].unit << std::endl;
```

Comment:
instead of relying on the user to provide values within the limits, can you add some checks here to make sure the values are reasonable before sending them to the driver?